### PR TITLE
Fix minor issues in UG, DG, index.md

### PR DESCRIPTION
**Pull request by ElginL**
- [User Guide] content ordering now ordered according to the summary list (links) at the top of the page.
- [User Guide] summary command at the bottom of the page table format fixed, and added some missing commands
- [Developer Guide] Fixed trivial grammar mistakes
- [index.md] Codecov and build status now links to our own one instead of the AddressBook's one

### CODE REVIEW

**Review by jicsontoh — APPROVED** [review]
LGTM!

File: docs/UserGuide.md [hunk 1]
```diff
@@ -21,11 +21,11 @@ interactions recorded faster and simpler than traditional GUI applications.
   * [Locating companies by name: `find`](#locating-companies-by-name-find)
   * [Viewing help: `help`](#viewing-help--help)
   * [Getting the User Guide: `user guide`](#getting-the-user-guide-user-guide)
+  * [Sorting the address book: `sort`](#sorting-the-address-book-sort)
+  * [Creating a transaction: `transaction`](#creating-a-transaction-transaction)
   * [Saving the data](#saving-the-data)
   * [Editing the data file](#editing-the-data-file)
   * [Archiving data files `coming in v2.0`](#archiving-data-files-coming-in-v20)
```
Comment (notle1706) [inline docs/UserGuide.md:28]: Is this supposed to be v2.0 instead?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -87,20 +87,6 @@ interactions recorded faster and simpler than traditional GUI applications.
...
-
-Examples:
-* `help` Shows the list of commands.
-* `help find` Shows the description of `find` command.
-* `help add` Shows the description of `add` command.
-
-
 ### Adding a person: `add`
 
 Adds a point of contact to the company.
```
Comment (notle1706) [inline docs/UserGuide.md]: Should we standardise it to point-of-contact instead?

File: docs/UserGuide.md [hunk 3]
```diff
@@ -154,47 +177,46 @@ Format: `find KEYWORD [MORE_KEYWORDS]`
 * Persons matching at least one keyword will be returned (i.e. `OR` search).
```
Comment (notle1706) [inline docs/UserGuide.md]: Should this be companies instead?

**Review by notle1706 — COMMENTED** [review]
Minor changes to be made.

File: docs/DeveloperGuide.md [hunk 4]
```diff
@@ -276,21 +276,21 @@ _{Explain here how the data archiving feature will be implemented}_
...
-| `* *`   | careless business owner   | edit the details of point-of-contact       | correct the mistakes that I did                                    |
-| `* *`   | efficient business owner  | sort the search result by price            | quickly know which company I have made the most transaction with    |
+| Priority | As a …​                  | I want to …​                                        | So that I can…​                                                     |
+|----------|--------------------------|-----------------------------------------------------|---------------------------------------------------------------------|
+| `* * *`  | business owner           | choose which company and point-of-contact to delete | save only relevant companies and point-of-contact to remove clutter |
+| `* * *`  | forgetful business owner | see usage instructions                              | refer to instructions when I forget how to use the application      |
+| `* * *`  | new business owner       | simply search for contacts                          | retrieve the only essential details i need                          |
```
Comment (notle1706) [inline docs/DeveloperGuide.md]: I think there's a grammatical error here. It should be "retrieve only the essential details I need."

File: docs/DeveloperGuide.md [hunk 5]
```diff
@@ -276,21 +276,21 @@ _{Explain here how the data archiving feature will be implemented}_
...
+| `* *`    | efficient business owner | sort the search result by price                     | quickly know which company I have made the most transaction with    |
 
 *{More to be added}*
 
 ### Use cases
 
-(For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
+(For all use cases below, the **System** is the `JeeqTracker` and the **Actor** is the `user`, unless specified otherwise)
 
 **Use case: Delete a person**
```
Comment (notle1706) [inline docs/DeveloperGuide.md]: Should this be Delete a company? Or are we not changing it yet? If we are, the lines below have to be changed as well.

**Review by notle1706 — COMMENTED** [review]
Some minor typos.

**Review by notle1706 — APPROVED** [review]
LGTM

**Pull request merged**